### PR TITLE
Feature/new wishlist and subscription events

### PR DESCRIPTION
**Pull request by elizabethbrady-ep**
Adding 3 new events per Dr. Cilabo new events:

remove_from_wishlist
wishlist_viewed
subscription_change

### CODE REVIEW

**Review by StevenJRowe123 — APPROVED** [review]
This all looks good to me. Thanks!

File: events/subscriptions/subscription_change.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Subscribe
```
Comment (tyssejc) [inline events/subscriptions/subscription_change.md:1]: change to **Subscription Change**

File: events/subscriptions/subscription_change.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Fire whenever a user changes their subscription (other than a subscribe or unsubscribe event).
+
+## HTML Data Attributes
+
+TBD
```
Comment (tyssejc) [inline events/subscriptions/subscription_change.md:7]: nit, but suggest leaving this out until we know which (if any) attributes will be required

File: events/ecommerce/wishlist_viewed.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+```js
+// When:
+// User user views their wishlist
+
+// Code:
+window.dataLayer = window.dataLayer || [];
+dataLayer.push({ ecommerce: null });  // Clear the previous ecommerce object.
+dataLayer.push({
+  event: "wishlist_viewed",
```
Comment (tyssejc) [inline events/ecommerce/wishlist_viewed.md:15]: suggest `view_wishlist` instead. consider other `view_` events in GA4 spec:
- `view_cart`
- `view_item`
- `view_item_list`
- `view_promotion`

regardless of how DCL implements, this should reflect what we want as canonized in the spec. @jcottriel-sdi thoughts?

**Review by tyssejc — CHANGES_REQUESTED** [review]
Thanks @elizabethbrady-ep ! I left a few comments.

File: events/subscriptions/subscription_change.md (hunk 1 above)
Comment (elizabethbrady-ep) [reply events/subscriptions/subscription_change.md]: done

File: events/ecommerce/wishlist_viewed.md (hunk 3 above)
Comment (StevenJRowe123) [reply events/ecommerce/wishlist_viewed.md:15]: I'm happy to change this around to view_whislist.

**Review by tyssejc — APPROVED** [review]
Looks good — thanks @elizabethbrady-ep ! Approved.

**Review by StevenJRowe123 — APPROVED** [review]

**Pull request merged**